Source/WebKit/Platform/cocoa/AssertionCapability.mm

```diff
@@ -39,17 +39,17 @@
     , m_willInvalidateBlock { makeBlockPtr(WTFMove(willInvalidateFunction)) }
     , m_didInvalidateBlock { makeBlockPtr(WTFMove(didInvalidateFunction)) }
 {
-}
-
-RetainPtr<_SECapability> AssertionCapability::platformCapability() const
-{
 #if USE(EXTENSIONKIT)
     if ([get_SECapabilityClass() respondsToSelector:@selector(assertionWithDomain:name:environmentIdentifier:willInvalidate:didInvalidate:)])
-        return [get_SECapabilityClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:environmentIdentifier() willInvalidate:m_willInvalidateBlock.get() didInvalidate:m_didInvalidateBlock.get()];
+        m_capability = [get_SECapabilityClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:m_environmentIdentifier willInvalidate:m_willInvalidateBlock.get() didInvalidate:m_didInvalidateBlock.get()];
     if ([get_SECapabilityClass() respondsToSelector:@selector(assertionWithDomain:name:environmentIdentifier:)])
-        return [get_SECapabilityClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:environmentIdentifier()];
+        m_capability = [get_SECapabilityClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:m_environmentIdentifier];
 #endif
-    return nil;
+}
+
+RetainPtr<_SECapability> AssertionCapability::platformCapability() const
```

Comment:
I think this should return ` _SECapability *`

Reply:
Would you be ok with keeping this as-is, since this is a virtual method, and changing this would be a bigger change?

Thanks for reviewing!

Reply:
Ok.

Reply:
Thank you Chris!